operator/internal/manifests/openshift/dashboards.go

```diff
@@ -0,0 +1,89 @@
+package openshift
+
+import (
+	"encoding/json"
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	monitoringv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+
+	"github.com/grafana/loki/operator/internal/manifests/openshift/internal/dashboards"
+)
+
+const (
+	labelConsoleDashboard  = "console.openshift.io/dashboard"
+	managedConfigNamespace = "openshift-config-managed"
```

Comment:
I feel like I'm missing something, do dashboards need to explicitly live in this namespace? If not why the difference in namespace?

Reply:
Yes openshift console dashboard live in this special namespace as ConfigMap resources.